### PR TITLE
Reducing the scope of variable after running static code analyser.

### PR DESCRIPTION
**Pull request by kushalsingh007**
- Part of fix for issue number 480 (variableScope)
- Suppression of warnings that are not needed.

### CODE REVIEW

**Comment by kushalsingh007** [comment]
Fix for variable Scope issue for #480

File: drivers/lis3dh/lis3dh.c [hunk 1]
```diff
@@ -64,7 +64,9 @@ int lis3dh_init(lis3dh_t *dev, spi_t spi, gpio_t cs_pin, gpio_t int1_pin, gpio_t
 
 int lis3dh_read_xyz(const lis3dh_t *dev, lis3dh_data_t *acc_data)
 {
+    /* cppcheck-suppress variableScope */
     int32_t tmp;
```
Comment (gebart) [inline drivers/lis3dh/lis3dh.c]: Move this inside the `for` loop on L88 instead of suppressing here.

File: sys/net/routing/nhdp/nib_table.c [hunk 2]
```diff
@@ -252,10 +252,11 @@ static void clear_nb_addresses(nib_entry_t *nib_entry, nhdp_addr_entry_t *nb_lis
                                nhdp_addr_entry_t **out_list, timex_t *now)
 {
     nhdp_addr_entry_t *elt, *nib_elt, *nib_tmp;
-    uint8_t found;
 
     LL_FOREACH_SAFE(nib_entry->address_list_head, nib_elt, nib_tmp) {
-        found = 0;
+        uint8_t found;
+	 found = 0;
```
Comment (gebart) [inline sys/net/routing/nhdp/nib_table.c]: Join the above two lines.

**Comment by gebart** [comment]
ACK when Travis is happy and my comments are addressed.

**Comment by kushalsingh007** [comment]
Please restart the build :)

**Comment by kushalsingh007** [comment]
restart needed yet again.

**Comment by gebart** [comment]
Travis is being more unreasonable that usual today...

**Comment by kushalsingh007** [comment]
All is well :smile:

**Comment by gebart** [comment]
& go!

**Pull request merged**